package.json

```diff
@@ -1,86 +1,87 @@
-  },
-  "license": "BSD-3-Clause",
-  "author": {
-    "name": "Josh Geden",
-    "email": "jgeden@lbl.gov"
-  },
-  "files": [
-    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
-    "style/**/*.{css,.js,eot,gif,html,jpg,json,png,svg,woff2,ttf}",
-    "schema/*.json",
-    "style/index.js"
-  ],
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
-  "style": "style/index.css",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/NERSC/nersc_refresh_announcements.git"
-  },
-  "scripts": {
-    "build": "jlpm run build:lib && jlpm run build:labextension:dev",
-    "build:labextension": "jupyter labextension build .",
-    "build:labextension:dev": "jupyter labextension build --development True .",
-    "build:lib": "tsc",
-    "build:prod": "jlpm run clean && jlpm run build:lib && jlpm run build:labextension",
-    "clean": "jlpm run clean:lib",
-    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
-    "clean:labextension": "rimraf nersc_refresh_announcements/labextension",
-    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
-    "eslint": "eslint . --ext .ts,.tsx --fix",
-    "eslint:check": "eslint . --ext .ts,.tsx",
-    "install:extension": "jlpm run build",
-    "watch": "run-p watch:src watch:labextension",
-    "watch:labextension": "jupyter labextension watch .",
-    "watch:src": "tsc -w"
-  },
-  "dependencies": {
-    "@jupyterlab/application": "^4.0.0",
-    "@lumino/widgets": "^2.0.1"
-  },
-  "devDependencies": {
-    "@jupyterlab/builder": "^4.0.0",
-    "@types/json-schema": "^7.0.11",
-    "@types/react": "^18.0.26",
-    "@typescript-eslint/eslint-plugin": "^5.55.0",
-    "@typescript-eslint/parser": "^5.55.0",
-    "css-loader": "^6.7.1",
-    "eslint": "^8.36.0",
-    "eslint-config-prettier": "^8.7.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.8.7",
-    "rimraf": "^4.4.1",
-    "source-map-loader": "^1.0.2",
-    "style-loader": "^3.3.1",
-    "stylelint": "^14.9.1",
-    "stylelint-config-prettier": "^9.0.4",
-    "stylelint-config-recommended": "^8.0.0",
-    "stylelint-config-standard": "^26.0.0",
-    "stylelint-prettier": "^2.0.0",
-    "typescript": "~5.0.2",
-    "yjs": "^13.5.40"
-  },
-  "sideEffects": [
-    "style/*.css",
-    "style/index.js"
-  ],
-  "styleModule": "style/index.js",
-  "jupyterlab": {
-    "extension": true,
-    "outputDir": "nersc_refresh_announcements/labextension",
-    "schemaDir": "schema"
-  }
+    "name": "stsci-announce",
+    "version": "0.5.0",
+    "description": "A JupyterLab extension.",
+    "keywords": [
+        "jupyter",
+        "jupyterlab",
+        "jupyterlab-extension"
+    ],
+    "homepage": "https://github.com/spacetelescope/stsci-announce",
+    "bugs": {
+        "url": "https://github.com/spacetelescope/stsci-announce/issues"
+    },
+    "license": "BSD-3-Clause",
+    "author": {
+        "name": "Josh Geden",
+        "email": "jgeden@lbl.gov"
```

Comment:
Do we need to add more authors here?

Reply:
That's a good point.   I don't think you can,  but we could add a maintainers section I believe and leave Josh as-is:   https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors.  The repo itself is already tracking contributors.